pkgs/misc/game-rs/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, steam-run
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "game-rs";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "amanse";
+    repo = "game-rs";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-M9/hFItoCL8fSrc0dFNn43unqkIaD179OGUdbXL6/Rs=";
+  };
+
+  buildFeatures = [ "nixos" ];
+
+  propagatedBuildInputs = [ steam-run ];
+
+  cargoLock = {
```

Comment:
As above, can we use cargoHash? https://nixos.org/manual/nixpkgs/unstable/#compiling-rust-applications-with-cargo

Reply:
ofcourse, I'll get right to it

Reply:
Done!